### PR TITLE
MultiStage AboutWelcome Experiment test Recipe

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: messaging-experiments.yaml [hunk 1]
```diff
@@ -165,3 +165,70 @@
                     string_id: onboarding-browse-privately-button
                   action:
                     type: OPEN_PRIVATE_BROWSER_WINDOW
+- id: MULTISTAGE_ABOUT_WELCOME_TEST
+  enabled: true
+  filter_expression: env.version >= '78.' && env.version < '79.' &&
```
Comment (ncloudioj) [inline messaging-experiments.yaml:170]: Do we still need the `filter_expression` now? I think this field is only required for 77, can you confirm?

File: messaging-experiments.yaml [hunk 2]
```diff
@@ -165,3 +165,70 @@
...
+                subtitle: Coming from another browser? It's easy to bring everything to Firefox.
+                tiles:
+                primary_button:
+                  label: Start Import
+                  action:
+                    navigate: true
+                    type: SHOW_MIGRATION_WIZARD
+                    addFlowParams: true
+                    data:
+                      args: https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead
```
Comment (ncloudioj) [inline messaging-experiments.yaml:229]: Do we still need this `data` section for the `SHOW_MIGRATION_WIZARD` special message action?

File: messaging-experiments.yaml (hunk 1 above)
Comment (punamdahiya) [reply messaging-experiments.yaml:170]: In 76 we had limitation of using filter_expression only. We are using filter_expression and targeting both 77 onwards,  filter_expression gives flexibility to use  Remote Settings filter where as targeting is limited to ASRoutertargeting

Comment (ncloudioj) [reply messaging-experiments.yaml:170]: okay. I find this a bit confusing, for example, the Firefox version targeting was specified in both `filter_expression` and `targeting`, and they are not different. The one in `targeting` seems a bit redundant and inconsistent to me.

File: messaging-experiments.json [hunk 3]
```diff
@@ -261,5 +261,92 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTISTAGE_ABOUT_WELCOME_TEST",
+    "enabled": true,
+    "filter_expression": "env.version >= '78.' && env.version < '80.'",
+    "targeting": "localeLanguageCode == \"en\" && firefoxVersion >= 78 && (!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue) || 'multistage_aboutwelcome_test' in activeExperiments)",
```
Comment (punamdahiya) [inline messaging-experiments.json:269]: Simplified targeting and filter expression for easier testing in local dev when pointed to stage

**Comment by punamdahiya** [comment]
> Thanks!
> 
> I will fix the validation for the new schema in a follow up.

Thank you!

**Pull request merged**